demo/app.js

```diff
@@ -2,11 +2,13 @@ import React from 'react'
 
 import TimeSeriesContainer from './ts-container'
 import BreakdownContainer from './b-container'
+import ScatterPieContainer from './sp-container'
 
 const App = () => (
   <React.Fragment>
-    <BreakdownContainer />
+    {false && <BreakdownContainer />}
```

Comment:
Just remove them altogether since now this demo is consolidated into `ScatterPieContainer` (but maybe rename it to `BreakdownContainer`)